solidity/contracts/bridge/Bridge.sol

```diff
@@ -1541,6 +1563,72 @@ contract Bridge is
         self.updateTreasury(treasury);
     }
 
+    /// @notice Allows authorized controllers to increase Bank balances via the
+    ///         Bridge.
+    /// @dev "Controllers" here are addresses that have been authorized via
+    ///      `setAuthorizedBalanceIncreaser` and stored in the
+    ///      `authorizedBalanceIncreasers` mapping.
+    /// @param recipient Address receiving the balance increase.
+    /// @param amount Amount by which the balance is increased.
+    function controllerIncreaseBalance(address recipient, uint256 amount)
+        external
+    {
+        require(
+            self.authorizedBalanceIncreasers[msg.sender],
+            "Caller is not an authorized increaser"
+        );
+        self.bank.increaseBalance(recipient, amount);
+    }
+
+    /// @notice Allows authorized controllers to increase multiple Bank
+    ///         balances via the Bridge.
+    /// @dev See `controllerIncreaseBalance` for the definition of a
+    ///      controller and how authorization is managed.
+    /// @param recipients Addresses receiving the balance increases.
+    /// @param amounts Amounts by which balances are increased.
+    function controllerIncreaseBalances(
+        address[] calldata recipients,
+        uint256[] calldata amounts
+    ) external {
+        require(
+            self.authorizedBalanceIncreasers[msg.sender],
+            "Caller is not an authorized increaser"
+        );
+        self.bank.increaseBalances(recipients, amounts);
+    }
+
+    /// @notice Batch helper returning authorization status for given addresses.
+    /// @param increasers Addresses to check for controller authorization.
```

Comment:
empty arrays are cool or not ? just for completeness: maybe add `s.
/// @param increasers Addresses to check. `